test-data/pretty_format_kotlin/NotPrettyFormattedFixedKtfmtGoogle.kt

```diff
@@ -0,0 +1,3 @@
+fun main(args: Array<String>) {
```

Comment:
Not too sure about this one, since we share some files between the two tests. Also I'd have to change some of the helper code to look in different directory based on the formatter. 

I'm happy to do it if you really insist.